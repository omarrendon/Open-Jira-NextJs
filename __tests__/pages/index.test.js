import { create } from "react-test-renderer";
import HomePage from "../../pages";

describe('INITIAL TEST FILE' , () => {
  it('First test', () => {
    const result = 4;

    expect(2+2).toBe(result);
  });
});

describe('Snapshot <HomePage>', () => {
  it('Should be take a snapshot of HomePage component', () => {
    const  homePage = create(<HomePage />).toJSON();
    expect(homePage).toMatchSnapshot();
  });;
 });
