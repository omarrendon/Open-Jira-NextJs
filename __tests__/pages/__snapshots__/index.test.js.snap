// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot <HomePage> Should be take a snapshot of HomePage component 1`] = `
<div
  className="MuiBox-root css-p8hoxb"
>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionSticky css-7e0s21-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
    >
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart MuiIconButton-sizeLarge css-184mzhy-MuiButtonBase-root-MuiIconButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="MenuOutlinedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
          />
        </svg>
      </button>
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-rmjmia-MuiTypography-root-MuiLink-root"
        href="/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
      >
        <h6
          className="MuiTypography-root MuiTypography-h6 css-18omlhr-MuiTypography-root"
        >
          OpenJira
        </h6>
      </a>
    </div>
  </header>
  <div
    className="MuiBox-root css-1ecxfl0"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-wg4xta-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18z51xc-MuiPaper-root-MuiCard-root"
        >
          <div
            className="MuiCardHeader-root css-185gdzj-MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
              >
                Pendientes
              </span>
            </div>
          </div>
          <div
            className="MuiBox-root css-kuuj3l"
          >
            <button
              className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-fullWidth MuiButtonBase-root  css-xsqe0f-MuiButtonBase-root-MuiButton-root"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="AddCircleOutlineIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
                  />
                </svg>
              </span>
              Agregar tarea
            </button>
          </div>
          <div
            className=""
            onDragOver={[Function]}
            onDrop={[Function]}
          >
            <div
              className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-ee958u-MuiPaper-root"
            >
              <ul
                className="MuiList-root MuiList-padding css-b84wkv-MuiList-root"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-wg4xta-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18z51xc-MuiPaper-root-MuiCard-root"
        >
          <div
            className="MuiCardHeader-root css-185gdzj-MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
              >
                En Progreso
              </span>
            </div>
          </div>
          <div
            className=""
            onDragOver={[Function]}
            onDrop={[Function]}
          >
            <div
              className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-ee958u-MuiPaper-root"
            >
              <ul
                className="MuiList-root MuiList-padding css-b84wkv-MuiList-root"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 css-wg4xta-MuiGrid-root"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-18z51xc-MuiPaper-root-MuiCard-root"
        >
          <div
            className="MuiCardHeader-root css-185gdzj-MuiCardHeader-root"
          >
            <div
              className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
            >
              <span
                className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
              >
                Completadas
              </span>
            </div>
          </div>
          <div
            className=""
            onDragOver={[Function]}
            onDrop={[Function]}
          >
            <div
              className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-ee958u-MuiPaper-root"
            >
              <ul
                className="MuiList-root MuiList-padding css-b84wkv-MuiList-root"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
